t/query/country_cities.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+require_rel '../../lib/query/country_cities'
+
+describe Query::CountryCities do
+  describe 'United States of America (Q30)' do
+    around { |test| VCR.use_cassette('CountryCities Q30', &test) }
+
+    let(:cities) { Query::CountryCities.new(id: 'Q30').data }
+
+    it 'should only list Los Angeles once' do
+      cities.select { |c| c.name == 'Los Angeles' }.count.must_equal 1
+    end
```

Comment:
My thinking was that the hierarchy is _'United States of America (Q30)'_  > _'should only list Los Angeles once'_ but I did consider it the way you've described, and even had it that way in one iteration. So that says to me 👍 for this change.